sympy/geometry/tests/test_polygon.py

```diff
@@ -402,19 +449,23 @@ def test_intersection():
         Segment(Point(1, 0), Point(0, 1))]
     assert poly1.intersection(Point(0, 0)) == [Point(0, 0)]
     assert poly1.intersection(Point(-12,  -43)) == []
-    assert poly2.intersection(Line((-12, 0), (12, 0))) == [Point(-5, 0), Point(0, 0),
-                                                           Point(S(1)/3, 0), Point(1, 0)]
+    assert poly2.intersection(Line((-12, 0), (12, 0))) == [Point(-5, 0),
+        Point(0, 0),Point(S(1)/3, 0), Point(1, 0)]
     assert poly2.intersection(Line((-12, 12), (12, 12))) == []
-    assert poly2.intersection(Ray((-3,4), (1,0))) == [Segment(Point(1, 0), Point(0, 1))]
-    assert poly2.intersection(Circle((0, -1), 1)) == [Point(0, -2), Point(0, 0)]
+    assert poly2.intersection(Ray((-3,4), (1,0))) == [Segment(Point(1, 0),
+        Point(0, 1))]
+    assert poly2.intersection(Circle((0, -1), 1)) == [Point(0, -2),
+        Point(0, 0)]
     assert poly1.intersection(poly1) == [Segment(Point(0, 0), Point(1, 0)),
         Segment(Point(0, 1), Point(0, 0)), Segment(Point(1, 0), Point(0, 1))]
     assert poly2.intersection(poly2) == [Segment(Point(-5, 0), Point(0, -4)),
-        Segment(Point(0, -4), Point(0, S(1)/5)), Segment(Point(0, S(1)/5), Point(S(1)/2, -S(1)/10)),
-        Segment(Point(0, 1), Point(-5, 0)), Segment(Point(S(1)/2, -S(1)/10), Point(1, 0)),
+        Segment(Point(0, -4), Point(0, S(1)/5)),
```

Comment:
This may have to align opening limiter `[` when conforming to [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation).
But I would rather suggest using hanging indentation like 
```
# Hanging indents should add a level.
foo = long_function_name(
    var_one, var_two,
    var_three, var_four)
```
if the opening line is long.

Reply:
I agree. So may I keep this layout for these line?

Reply:
> when conforming to [PEP-8](https://www.python.org/dev/peps/pep-0008/#indentation).

I think we prefer not to follow this aspect of PEP8, so the lines can be kept as they are.

Reply:
(as you have made them)

Reply:
OK, Thanks a lot for reviewing my PR!